prog/vm/github_runner.rb

```diff
@@ -55,11 +55,11 @@ def self.pick_vm(label, project)
         amount: 1
       )
 
-      puts "Pool is used for #{label}"
+      puts "#{project} Pool is used for #{label}"
```

Comment:
should this be project ubid?

Reply:
Our models have an overridden string representation. It will be printed as `Project[UBID]`
https://github.com/ubicloud/ubicloud/pull/625